### DISCUSSION TITLE
DNSMASQ 禁用IPV6解析不生效

### DISCUSSION THREAD

**Discussion by devnakx** [Boos4721/openwrt#80] [Bugs/问题反馈]
![1](https://user-images.githubusercontent.com/38558065/156573012-71ea2b12-ebbf-44ec-aa58-b3805a1c00a7.png)

**Comment by Boos4721** [comment]
FULL LOG

**Reply by devnakx** [reply]
在系统日志里没有发现疑似异常的日志，而且我把`记录查询日志`勾上，也没有查询日志被写入系统日志

**Comment by Boos4721** [comment]
**FIXED**

**Reply by devnakx** [reply]
好的感谢
